__tests__/app.test.js

```diff
@@ -210,9 +249,24 @@ describe("GET /api/articles", () => {
 
         });
     });
-  });
-
-});
+    test("200: should give an array of all articles if given topic doesn't exist", () => {
+      return request(app)
+        .get("/api/articles?topic=habbory")
+        .expect(200)
+        .then(({ body: { articles } }) => { 
+          expect(articles.length).toBeGreaterThan(0);
+        })
+    });
```

Comment:
This is not a valid response, this should in fact be a 404 as the topic does not exist with an appropriate message